CMakeLists.txt

```diff
@@ -61,13 +61,13 @@ file(WRITE ${CMAKE_BINARY_DIR}/coverage_targets.txt "")
 # Determine platform to build
 if(AZ_PLATFORM_IMPL STREQUAL "WIN32")
   # build windows platform
-  set(PAL az_win32)
+  set(PORTS_INCLUDE_DIR win32)
```

Comment:
does our CMake no longer automatically decide it is on `posix` or `win32`

Reply:
The decision was made in the working group to select the _template folder unless a platform is specified.  This way those that need a platform will get build failures if they don't explicitly specify their target platform